docs/ui/workspaces.md

```diff
@@ -97,4 +97,36 @@ Select **Send** to initiate the invitation.
 
 To add a service account to a workspace, select the Service Accounts **+** icon. You can select from a list of existing service accounts configured for the organization. Select a Workspace Role for the service account. This will be the initial role for the service account within the workspace. A workspace Owner can change this role at any time. Select **Share** to finalize adding the service account.
 
-To delete a workspace collaborator or service account, select **Remove** from the menu on the right side of the user or service account information on this page.
\ No newline at end of file
+To delete a workspace collaborator or service account, select **Remove** from the menu on the right side of the user or service account information on this page.
+
+## Workspace transfer
+
+Workspace transfer enables you to move an existing workspace from a personal account to an organization, or between organizations.
+
+Transferring a workspace enables you to take existing configuration and flow run history with you as projects move to different teams or organizations. Without transfer, users moving to a new organization from, for example, a personal account would need to re-create all of their block, deployment, notification, and work queue configuration in a new workspace. Additionally, any flow run history would remain in the personal workspace. 
+
+!!! note "Workspace transfer permissions"
+    You must initiate workspace transfer from the workspace to be transferred. Therefore, workspace transfer must be initiated by a user with admin priviliges for the workspace to be transferred.
```

Comment:
in the near future (this week) users will have access to a new route app.prefect.cloud/transfer, which I think changes this messaging just slightly. The permissions piece is still true, but they can "begin" the transfer process without first entering the origin workspace page

Reply:
I think it makes sense to revise this.